src/patterns/task-list-pages/index.md

```diff
@@ -56,22 +51,16 @@ Where possible, task names should:
 
 ### Show the status of the tasks
 
-Include a summary above the task list to say how many tasks have been completed.
-
-You might find a better word to describe what a ‘task’ is for your users. If your service is an application form, each task might be a ‘section’ of questions to complete.
-
-![A screenshot showing a task list summary that says 'Application incomplete. You've completed 3 of 8 sections'](task-list-summary.png)
-
-Make it clear to users which tasks they’ve completed and which still need their attention, by labelling them using the [Tag component](/components/tag/).
+Make it clear to users which tasks they’ve completed and which still need their attention, by labelling them using statuses.
```

Comment:
We’ve added a lot more guidance on task list status within the component guidance page - including the suggestion that services start with just 2 statuses ("Completed" and "Incomplete") but could use 3 ("Completed", "Not yet started" and "In progress") if they find a need for it, or use custom statuses if they have additional needs.

We’ll need to decide whether to duplicate some of that content here, or only have this on either the component or pattern pages, with a link from the other one?

Any thoughts @stevenjmesser @Ciandelle @calvin-lau-sig7?

Reply:
@stevenjmesser ok, I've moved the status guidance to the pattern page, and just left a short stub about statuses on the component page (with another link to the pattern page). It's not perfect yet, but have a look and let me know what you think...